data/campaigns/Under_the_Burning_Suns/scenarios/02_Across_the_Harsh_Sands.cfg

```diff
@@ -1553,7 +1553,14 @@
                             [effect]
                                 apply_to=attack
                                 range=melee
-                                set_type=arcane
+                                [set_specials]
+                                    mode=append
+                                    [damage_type]
+                                        id=arcane_damage
+                                        type=arcane
+                                        cumulative=yes
+                                    [/damage_type]
+                                [/set_specials]
```

Comment:
Although this is a method of fixing bug #7774, the code now in #7775 is a simpler way to do it. Either this code in UtBS S02 needs to know about the event in S03, or that event needs to know about this event; however doing fixing it in S03 just needs a
```
[not]
    type=arcane
[/not]
```

Reply:
excuse me, i added https://github.com/wesnoth/wesnoth/pull/7003/commits/5e176e58da98aeda4d0b45a9c5019aab03e0dfea for fixing that. I don't think of https://github.com/wesnoth/wesnoth/issues/7774 issue when i postted https://github.com/wesnoth/wesnoth/pull/7003/commits/fd270c6cf0fd3afd7cbf3b8650ea6d8962292416

Reply:
Would this special still be active even if the primary type of the attack was changed? I expected that it would, therefore the code in #7775 would change a (blade + arcane) weapon into a (fire + arcane) weapon, even without fd270c6.

Reply:
Yes, it would remain active since I didn't implement a type-based filter, so even if I hadn't added a pure S3 secondary_type arcane filter, we would have fire+arcane.

Reply:
I added the filter in S3 with https://github.com/wesnoth/wesnoth/commit/5e176e58da98aeda4d0b45a9c5019aab03e0dfea because it is the blessing of an ordinary weapon that competes with holy water, and one should have either fire or arcane but not fire+arcane, and therefore incase already blessed weapons, the fire blessing should not apply.